gen/go/proto/v1/config.proto

```diff
@@ -218,12 +218,16 @@ message EndpointMetricsConfig {
 }
 
 message MetricsConfig {
-    // set this flag to enable metrics
-    google.protobuf.BoolValue enabled = 1;
-    // endpoint level configuration
-    EndpointMetricsConfig endpoint_config = 2;
-    // config for printing metrics in logs
-    MetricsLogConfig logging = 3;
+  // set this flag to enable metrics
```

Comment:
@tim-mwangi @ryanericson  I was trying to merge this with the existing telemetry config. But can't think of a good way as libtraceable configuration has all these subconfigurations

Reply:
What are all the subconfigurations?

Reply:
The main one is a logging configuration which prints the stats. Then there is an endpoint level configuration which prints the stats at an endpoint level. Then I was thinking of adding a reporter/exporter configuration in this pr so that we can turn on/off libtraceable's metrics separately from the agent's metrics